state-chain/pallets/cf-elections/src/electoral_systems/block_witnesser/state_machine.rs

```diff
@@ -104,6 +91,7 @@ pub trait BWProcessorTypes: Sized {
 )]
 pub struct BlockWitnesserSettings {
 	pub max_concurrent_elections: u16,
+	pub safety_margin: u32,
```

Comment:
This will default to 0. We should add a migration that inserts the correct safety margin on upgrade.